poetry.lock

```diff
@@ -576,16 +576,19 @@ test = ["pexpect"]
 
 [[package]]
 name = "jupyter-core"
-version = "4.9.2"
+version = "4.11.2"
```

Comment:
Do we even need jupyter in hokusai? We don't make notebooks in there right? what is jupyter-core for here? @artsy/velocity-engineers